### PR TITLE
use timeseriesQuery for scalyr.count()

### LINKED ISSUE #213: Switch Scalyr wrapper count to timeseriesQuery
Switching to ``timeseriesQuery`` from ``numericQuery`` is considered more efficient, as recommended by Scalyr.

We will still keep the ``count`` method, only the internals will change.

### PR DESCRIPTION
**Pull request by vetinari**
as suggested by scalyr to get a faster result

fixes #213

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: zmon_worker_monitor/builtins/plugins/scalyr.py [hunk 1]
```diff
@@ -37,26 +37,30 @@ def __init__(self, read_key):
         self.read_key = read_key
 
     def count(self, query, minutes=5):
```
Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/scalyr.py]: Can't we just call ``timeseries()`` internally?

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Comment by vetinari** [comment]
:+1:

**Pull request merged**